### PR TITLE
Permit Card Updates

### PR DESCRIPTION
**Pull request by sanjaytkbabu**
# Description
[PADS-21](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-21) 
New StatusPill component
Permit card CSS updates

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/common/StatusPill.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+<template>
+  <div class="col-1">
+    <div :class="['authIndicator', toKebabCase(authStatus)]">
+      {{ authStatus }}
+    </div>
+  </div>
+</template>
+<style scoped lang="scss">
+.authIndicator {
```
Comment (jujaga) [inline frontend/src/components/common/StatusPill.vue:18]: Style: IIRC best practices for CSS element naming should follow BEM naming conventions. In this case, this class should be kebab cased to `.auth-indicator`.
- Ref: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

File: frontend/src/components/permit/PermitCard.vue [hunk 2]
```diff
@@ -158,15 +165,25 @@ p {
...
+  }
+}
+
+.completed {
+  background-color: #05366210;
 }
 
 .darkgrey {
-  color: darkgrey;
+  color: #888888;
```
Comment (jujaga) [inline frontend/src/components/permit/PermitCard.vue:187]: UX: This doesn't look like a standard BC Grayscale color. Please consult the BC design system for prescribed colors and verify with UX.
- Ref: https://www2.gov.bc.ca/gov/content/digital/design-system/foundations/colour - Greyscale section. Quick inspection leads me to believe that the closest equivalent color would be `theme.gray70`, but you'd have to verify with UX on that.

Style: This should be something we should be pulling from our global stylesheet if possible in lieu of a specific component SCSS override if possible. May need to be revisited in the future depending on timelines.

File: frontend/src/components/permit/PermitCard.vue [hunk 3]
```diff
@@ -158,15 +165,25 @@ p {
...
     padding-bottom: 0;
   }
+
+  :deep(.p-card-body) {
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 16px;
+    border-color: #05366260;
+    box-shadow: 0px 4px 4px 0px #0000001a;
+  }
```
Comment (jujaga) [inline frontend/src/components/permit/PermitCard.vue:179]: UX: Double check the guidance w.r.t. sizing. IIRC the industry prefers to lean towards relativistic measurement units like em or rem in lieu of px, especially when considering high DPI monitors.

**Comment by kyle1morel** [comment]
Text is not appearing centered in the status pill

File: frontend/src/components/common/StatusPill.vue (hunk 1 above)
Comment (sanjaytkbabu) [reply frontend/src/components/common/StatusPill.vue:18]: done, thank you!

File: frontend/src/components/permit/PermitCard.vue (hunk 2 above)
Comment (sanjaytkbabu) [reply frontend/src/components/permit/PermitCard.vue:187]: UX suggested this colour to improve visibility of 'Last updated' in the Permit card.

**Comment by sanjaytkbabu** [comment]
> Text is not appearing centered in the status pill

done

File: frontend/src/components/common/StatusPill.vue [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+// Props
+const { authStatus } = defineProps<{
+  authStatus?: string;
+}>();
+</script>
+
+<template>
+  <div class="mr-3 flex justify-content-center">
+    <div :class="['align-items-center', 'auth-indicator', toKebabCase(authStatus)]">
```
Comment (kyle1morel) [inline frontend/src/components/common/StatusPill.vue:12]: Please split static and dynamic classes per Discord conversation

**Review by jujaga — APPROVED** [review]

**Review by kyle1morel — APPROVED** [review]

**Pull request merged**